main.go

```diff
@@ -18,6 +17,10 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/tarball"
 
 	cobra "github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1alpha1 "kubevirt.io/api/export/v1alpha1"
```

Comment:
Please use v1beta1 https://github.com/kubevirt/kubevirt/tree/main/staging/src/kubevirt.io/api/export/v1beta1

Reply:
Ah good catch. I just copied-pasted these deps from other project, because I had lots of deps errors when simply I've added new kubevirt's deps.

Reply:
Done.